### ISSUE TITLE
How to add more tools?

### ISSUE THREAD

**Issue by FBurkelwsd** [packofparts/ehsshop#2]
If I wanted to add more of our tools to your awesome website then how would I enter information -and inmates? 

I see you have committed items.

**Comment by geeknoid** [comment]
There are two ways to add content:

* Using the GitHub UI
* Using the git command (what I do)

I'll assume you'll want to use the GitHub UI. So:

* You need to navigate in GitHub to the folder where to add a file.
* Select "Add files" from the top right of the window
* Enter the content in the editor and save it.
* After saving, the website will get automatically recreated and published within a few minutes.

The text is written in something called "markdown files". You'll see those in the website with a .md file extension. If you search online, you can find many tutorials explained how to use markdown, but it's pretty simple overall. Look at the existing .md files to see how it fits together.

When you add new content, you need to also add an entry for the new page to the table of contents.

**Comment by FBurkelwsd** [comment]
This does not seem to be saved into the content tools folder section. Working on adding something about sanding.
Does it need to be approved? also ran into an issue with uploading images. Made a pull request as I was trying out various features. 
Have also attempted to edit the md file to add a folder for "sanding"

**Comment by geeknoid** [comment]
Sorry, I didn't see the pull request you created. 

Yes, after you open a pull request, it needs to be approved in order to be merged into the repo. Once it is merged, then a few minutes later the web site will be automatically updated.

Regarding the file you submitted, you need to put a bit more stuff in the markdown file. If you look at the other files on the site, you'll see a few things. For example:

```
---
title: Band Saw
---

![Delta Band Saw](./band-saw.webp)

A band saw has a long, sharp blade consisting of a continuous band of toothed metal stretched between two or more wheels to cut material. They are used principally in woodworking, metalworking, and lumbering, but may cut a variety of materials. Advantages include uniform cutting action as a result of an evenly distributed tooth load, and the ability to cut irregular or curved shapes like a jigsaw.[1] The minimum radius of a curve is determined by the width of the band and its kerf. Most bandsaws have two wheels rotating in the same plane, one of which is powered, although some may have three or four to distribute the load. The blade itself can come in a variety of sizes and tooth pitches (teeth per inch, or TPI), which enables the machine to be highly versatile and able to cut a wide variety of materials including wood, metal and plastic. Band saw is recommended for use in cutting metal as it produces much less toxic fumes and particulates when compared with angle grinder and reciprocating saw.

The shop has two [Delta](./Delta_Band_Saw.pdf) band saws.

## Safety

You may learn more by reading this [Band Saw Safety Guide](https://www.powertoolinstitute.com/pti-includes/pdfs/Tool-Specific-Files/Band-Saws.pdf) and
watching this [Power Tool Safety](http://www.powertoolinstitute.com/pti-pages/videos/Power-Tool-Safety-Video-2018/index.html) video.

## Useful Links

* [Product Manual](./Delta_Band_Saw.pdf)
* [Learn More About Band Saws](https://en.wikipedia.org/wiki/Band_saw) from Wikipedia
* [How to Use a Bandsaw](https://www.youtube.com/watch?v=6XPK9xJyEPs) video from Fix This Build That
```

* First at the top is the title of the page, which requires the --- around it.
* Next, there are section headers which start with `##` such as `## Safety`.
* Next, see the way links are specified in the format `[Text To Display](link to go to)`.

So that's about how to write markdown files. Here's a site that explains the full capabilities of markdown: https://www.markdownguide.org/getting-started/.

Next, the image you want needs to be uploaded as a file that's part of the pull request and not as an image you paste into the description of the pull request. I think GitHub has an "add file" option to let you do this.

Next, the markdown file you create must have a link to the image file you're uploading. That's what the first line until the title in the above example is doing. It inserts the image mentioned into the page.

And finally, in order for the page to be discoverable when you navigate the website, you need to update the _index.md files to list the new page.

I know this sounds like a lot, but it's actually reasonably simple once you get the hang of it. Millions of sites are built with markdown in this way, it's a pretty common thing nowadays so there are a lot of references available on the Internet for this.
